sktime/tests/_config.py

```diff
@@ -244,6 +244,11 @@
         "test_fit_idempotent",
     ],
     "TSRGridSearchCV": ["test_multioutput"],  # see 6708
+    # pickling problem
+    "ChronosForecaster": [
+        "test_persistence_via_pickle",
+        "test_save_estimators_to_file",
+    ],
```

Comment:
do you know why its failing to save the model?

Are you receiving the same error while saving the model?
```
AttributeError: Can't get local object 'add_hook_to_module.<locals>.new_forward'
```